src/TokenProviders/Database/index.ts

```diff
@@ -33,6 +33,13 @@ export class TokenDatabaseProvider implements TokenProviderContract {
 			: this.db.connection()
 	}
 
+	/**
+	 * Returns the User's identifier column
+	 */
+	private getForeignKey() {
+		return this.config.foreignKey ?? 'user_id'
```

Comment:
Maybe this can be `this.config.foreignKey || 'user_id'`? The `??` compiles to the following

```
this.config.foreignKey !== null && this.config.foreignKey !== void 0 ? this.config.foreignKey : 'user_id';
```

It means a empty will pass the check and foreign key will be set to a empty string